apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -1,25 +1,72 @@
-<p
-  class="font-title text-[28px] font-medium mt-8 mb-5 text-title text-center sm:mt-12 sm:mb-[-22px] sm:text-left"
-  translate
->
-  record.metadata.api
-</p>
-<gn-ui-carousel
-  containerClass="gap-4 py-10"
-  stepsContainerClass="right-[var(--container-outside-width)] bottom-[calc(100%-12px)] top-auto"
->
-  <gn-ui-api-card
-    *ngFor="
-      let link of facade.apiLinks$ | async;
-      let first = first;
-      let last = last
-    "
-    [link]="link"
-    class="w-80"
-    [ngClass]="{
-      'mr-[var(--container-outside-width)]': last,
-      'ml-[var(--container-outside-width)]': first
-    }"
+<div class="flex flex-row">
+  <p
+    class="font-title text-[28px] font-medium mt-8 mb-5 text-title text-center sm:mt-12 sm:mb-[-22px] sm:text-left"
+    translate
   >
-  </gn-ui-api-card>
-</gn-ui-carousel>
+    record.metadata.api
+  </p>
+  <mat-tab-group
+    (selectedIndexChange)="onTabIndexChange($event)"
+    animationDuration="200ms"
+    mat-stretch-tabs="false"
+    mat-align-tabs="start"
+  >
+    <mat-tab>
+      <ng-template mat-tab-label>
+        <span class="tab-header-label sm:ml-32" translate
+          >URLS PREDEFINIES</span
+        >
+      </ng-template>
+      <div class="block py-10">
+        <gn-ui-carousel
+          containerClass="gap-4 py-10"
+          stepsContainerClass="right-[var(--container-outside-width)] bottom-[calc(100%-12px)] top-auto"
+        >
+          <gn-ui-api-card
+            *ngFor="
+              let link of facade.apiLinks$ | async;
+              let first = first;
+              let last = last
+            "
+            [custom]="false"
+            [link]="link"
+            class="w-80"
+            [ngClass]="{
+              'mr-[var(--container-outside-width)]': last,
+              'ml-[var(--container-outside-width)]': first
+            }"
+          >
+          </gn-ui-api-card>
+        </gn-ui-carousel>
+      </div>
+    </mat-tab>
+    <mat-tab>
```

Comment:
Too much duplicated code.
Probably a way to handle that with template outlet + context.
Anyway, don't jump into that yet cause I don't see the point of those 2 tabs, wait Lucile to give feedbacks.